docs/includes/eloquent-models/PlanetSchemaVersion1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Models;
+
+use MongoDB\Laravel\Eloquent\Model;
+
+class Planet extends Model
```

Comment:
The idea behind "Schema Versioned" is that the data model changes over time. So you should explain that the developer initially created this data model an used data in production, and later they needed to update the model (v2), so they can use the `HasSchemaVersion` trait to migrate the data when they are retrieved instead (or before) running a full migration of the database.

Reply:
I see, I will make some edits to the docs. In this model, does `galaxy` need to be fillable? It is not one of the fields in the original data schema

Reply:
`galaxy` don't need to be fillable in any of the model versions, as it's not used with `::create`.